package.json

```diff
@@ -34,5 +34,22 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@babel/core": "^7.19.0",
+    "@babel/eslint-parser": "^7.18.9",
+    "@babel/plugin-syntax-jsx": "^7.18.6",
+    "@babel/preset-env": "^7.19.0",
+    "@babel/preset-react": "^7.18.6",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-react": "^7.31.8",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
   }
 }
```

Comment:
![Screenshot from 2022-09-14 11-44-23](https://user-images.githubusercontent.com/93039359/190080360-ef9bddee-d29a-4ca9-87c1-abcce8e2d88e.png)


- [ ] Your readme is not descriptive enough. Please add descriptive readme using [microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) :blush: Your readme is the documentation of your project so please take it serioulsy :blush:

Reply:
Okay, thank you. I will walk through it.